qa/standalone/ceph-helpers.sh

```diff
@@ -1354,6 +1355,7 @@ function get_timeout_delays() {
     $trace && shopt -u -o xtrace
     local timeout=$1
     local first_step=${2:-1}
+    local max_timeout=${3:-$MAX_TIMEOUT}
```

Comment:
@ErwanAliasr1 Nobody is passing the new parameter yet to get_timeout_delays() or MAX_TIMEOUT isn't > 0, so the changes to it aren't used yet.  (I'm in meetings this week, but will continue to review in between)

Reply:
@dzafman That's true. That's a pending change. I used it during my tests but didn't pushed it. If we get an agreement on its usage, I vote for setting MAX_TIMEOUT to 10 or 20 to avoid the 100sec delays.